automated-comments/java/two-fer/avoid_string_format.md

```diff
@@ -0,0 +1,7 @@
+Although `String.format` is a wonderful and powerful method, it is extremely
+slow for a problem that only requires simple String concatenation with the `+`
+operator. For more information about the performance of `String.format`, check
+out
+[Java String Concatenation: Which Way Is Best?](https://redfin.engineering/java-string-concatenation-which-way-is-best-8f590a7d22a8).
+
+Try again using simple and efficient String concatenation.
```

Comment:
```suggestion
Use simple and efficient `String` concatenation.
```

But perhaps this can be left out entirely.

Reply:
Maybe "Use efficient `String` concatenation." which leaves "simple" out of it.

Reply:
```suggestion
Use `String` concatenation.
```